src/ValuedTextInput/index.jsx

```diff
@@ -91,156 +93,132 @@ ValuedTextInput.propTypes =
     /**
      *  Placeholder text
      */
-    placeholder           : PropTypes.string,
+    placeholder        : PropTypes.string,
     /**
      *  HTML attribute controlling input spell check
      */
-    spellCheck            : PropTypes.bool,
+    spellCheck         : PropTypes.bool,
     /**
      *  Input text alignment
      */
-    textAlign             : PropTypes.oneOf( [ 'auto', 'left', 'right' ] ),
+    textAlign          : PropTypes.oneOf( [ 'auto', 'left', 'right' ] ),
     /**
      *  Input string value
      */
-    value                 : PropTypes.string,
+    value              : PropTypes.string,
     /**
      * Value label text
      */
-    valueLabel            : PropTypes.string,
+    valueLabel         : PropTypes.string,
     /**
      * Position of the value label
      */
-    valueLabelPosition    : PropTypes.oneOf( [ 'left', 'right' ] ),
+    valueLabelPosition : PropTypes.oneOf( [ 'left', 'right' ] ),
 };
 
 ValuedTextInput.defaultProps =
 {
-    className             : undefined,
-    cssMap                : undefined,
-    defaultValue          : undefined,
-    errorMessage          : undefined,
-    errorMessageIsVisible : false,
-    errorMessagePosition  : 'top',
-    forceHover            : false,
-    hasError              : false,
-    id                    : undefined,
-    inputRef              : undefined,
-    isDisabled            : false,
-    isReadOnly            : false,
-    label                 : undefined,
-    labelPosition         : 'top',
-    name                  : undefined,
-    onBlur                : undefined,
-    onChange              : undefined,
-    onClick               : undefined,
-    onFocus               : undefined,
-    onKeyDown             : undefined,
-    onKeyPress            : undefined,
-    onKeyUp               : undefined,
-    onMouseOut            : undefined,
-    onMouseOver           : undefined,
-    placeholder           : undefined,
-    spellCheck            : undefined,
-    textAlign             : 'auto',
-    valueLabel            : undefined,
-    valueLabelPosition    : 'left',
+    className          : undefined,
+    cssMap             : undefined,
+    defaultValue       : undefined,
+    forceHover         : false,
+    hasError           : false,
+    id                 : undefined,
+    inputRef           : undefined,
+    isDisabled         : false,
+    isReadOnly         : false,
+    name               : undefined,
+    onBlur             : undefined,
+    onChange           : undefined,
+    onClick            : undefined,
+    onFocus            : undefined,
+    onKeyDown          : undefined,
+    onKeyPress         : undefined,
+    onKeyUp            : undefined,
+    onMouseOut         : undefined,
+    onMouseOver        : undefined,
+    placeholder        : undefined,
+    spellCheck         : undefined,
+    textAlign          : 'auto',
+    value              : undefined,
+    valueLabel         : '%',
```

Comment:
I think this should remain `valueLabel: undefined`; `"valueLabel": "%"` is defined as an example in `defaults.json` (which is good), while this should be up to user to define (same as other `label` props)

Reply:
Well, when it's defined in `defaults.json` it doesn't work that's why I defined in both files to discuss about it.